packages/grafana-toolkit/docker/grafana-plugin-ci-alpine/scripts/deploy.sh

```diff
@@ -10,7 +10,9 @@ mkdir /lib64 && ln -s /lib/libc.musl-x86_64.so.1 /lib64/ld-linux-x86-64.so.2
 rm /bin/cp
 mv /usr/local/bin/cp /bin/cp
 
-apk add --no-cache curl 'nodejs-current=14.5.0-r0' npm yarn build-base openssh git-lfs perl-utils coreutils python3
+apk add --update nodejs
+apk add --update npm
```

Comment:
You should add these two to the command on line 15, because the `--no-cache` flag is vital. Also, I think updating packages when building a Docker image is considered an anti-pattern, due to non-determinism.